projects/storefrontapp-e2e-cypress/cypress/integration/smoke/outlets.e2e-spec.ts

```diff
@@ -73,16 +73,4 @@ context('Outlets', () => {
       });
     });
   });
-
-  it('should work on cxOutlet input template change', () => {
```

Comment:
Due to unregistering outletRef on its destroy, this test was not working, becasue it was based on changed routes but remained outletRefs. Now moved to unit test of outletRef